config/initializers/dotenv.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+require 'dotenv'
```

Comment:
You could also include this at the top of application.rb (either way).

Reply:
Umm i kept getting an error when putting it there but i may need to change some of the autoloading settings